xml/System.IO.Enumeration/FileSystemEnumerator`1.xml

```diff
@@ -256,7 +256,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Resets the enumerator to the initial position.</summary>
         <remarks>To be added.</remarks>
```

Comment:
Remarks should explain why this method is not supported.

Reply:
This is actually a very good question. @JeremyKuhne can you please tell us why the Reset method is not supported? In the code, it simply throws but there's no description why:

https://github.com/dotnet/corefx/blob/8e95261ef710e0df980a6d46923eb3c0e8b0bc83/src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerator.cs

There's also no mention of this behavior in your blog post:

https://blogs.msdn.microsoft.com/jeremykuhne/2018/03/09/custom-directory-enumeration-in-net-core-2-1/

Reply:
Ok I asked Jeremy and the reason is that we do not have to implement Reset when inheriting from IEnumerator, and this is explicitly described in the IEnumerator remarks. I'll add the remark too.
https://docs.microsoft.com/en-us/dotnet/api/system.collections.ienumerator?view=netcore-3.0